### ISSUE TITLE
App crashes when using React Native Debugger

### ISSUE THREAD

**Issue by Toastbroad** [software-mansion/react-native-reanimated#2819]
### Description

Due to [this line](https://github.com/software-mansion/react-native-reanimated/blob/9c441ac8050e7d389563b6281a533666cf70cc02/src/reanimated2/core.ts#L388) introduced [here](https://github.com/software-mansion/react-native-reanimated/pull/2761) the app crashes when using React Native Debugger since `global._chronoNow` is `undefined`. 

Error message is:  `global.performance.now is not a function`

Compare to: https://github.com/software-mansion/react-native-reanimated/issues/2760

### Changes

I would suggest a patch similar to the following:

```
global.performance = {
    now: global._chronoNow || (() => {}),
};
```

### Package version
- React Native Reanimated: 2.3.1
- React Native Debugger: 0.12.1

**Comment by mjmasn** [comment]
@Toastbroad I think this one is already fixed (see recent commits) - hopefully there will be a new release soon...

**Comment by NoahKr** [comment]
I'm experiencing the same issue.

**Comment by 29er** [comment]
me too. same issue. if i go into core.ts and change 
     ```
 global.performance = {
        now: global._chronoNow,
      };
```  to 
      
```
      if (global.performance == null) {
        global.performance = {
          now: global._chronoNow,
        };
      }
```

**Comment by piaskowyk** [comment]
Duplicate: https://github.com/software-mansion/react-native-reanimated/issues/2827
This issue is already resolved here: #2771 but it isn't released yet. We will release 2.3.2 soon, please be patient 🙏

**Comment by Biplovkumar** [comment]
> Duplicate: #2827 This issue is already resolved here: #2771 but it isn't released yet. We will release 2.3.2 soon, please be patient 🙏

will wait till 2.3.2.
